README.md

```diff
@@ -39,6 +53,52 @@ Our last security audit was on July 31, 2015 by NCC ([results](docs/resources/nc
 
 Any security vulnerabilities can be reported to security@docker.com.
 
+### Comparative Analysis to other signing systems
+
+#### Features 
+
+Signing Mechanism | Provenance | Delegation | Key Rotation | Freshness | Thresholds
+--- | --- | --- | --- | --- | ---
+TUF | ✔ | ✔ | ✔ | ✔ | ✔  
+Notary | ✔ | ✔ | ✔ | ✔ | Partial
+Traditional Package Signing | ✔ | ❌  | ❌  | ❌  | ❌  
+rkt trust | ✔ | ❌  | ❌  | ❌  | ❌  
+Red Hat Container Image Signing | ✔ | ❌  | ❌  | ❌  | ❌  
+YUM | ✔ | ❌  | ❌  | ❌  | ❌  
+APT | ✔ | ❌  | ❌  | ❌  | ❌  
+PyPI | ✔ | ❌  | ❌  | ❌  | ❌  
+GPG | ✔ | ❌  | ❌  | ❌  | ❌  
+No Signing | Partial (TLS) | ❌  | ❌  | ❌  | ❌  
+
+#### Protection from attacks
+
+Definitions taken from the [TUF specification](https://github.com/theupdateframework/tuf/blob/develop/docs/tuf-spec.md):
+
+- Arbitrary installation attacks. An attacker installs anything they want on the client system. That is, an attacker can provide arbitrary files in response to download requests and the files will not be detected as illegitimate.
```

Comment:
could we bold the attack names and link the table column headers back to these definitions?